### ISSUE TITLE
Sketch Export

### ISSUE THREAD

**Issue by iduuck** [koral/koral-server#3]
I just release a module that is able to "parse/export" Sketch files: [`sketchjs`](https://github.com/iDuuck/sketchjs). It's based upon `sketchtool` by BohemianCoding.
I think this would also be a nice fit for the `koral-server`, when there are multiple artboards, that we can export and preview all of them.

**Comment by gabrielecirulli** [comment]
Cool! Yeah that might be very useful. Does it have any OS dependencies (i.e. does it work on any system or only OS X)?

**Comment by iduuck** [comment]
I don't think so, since it was originally intented for Windows and Linux developers to export the .sketch files although they are not on a mac.

I think also Zeplin and other tools like Invision are using Sketchtool for there export.

**Comment by gabrielecirulli** [comment]
Sounds like it's gonna work then! :ok_hand:
